src/Icons/CheckmarkOutline.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+interface Props {
+  width?: string | number
+  height?: string | number
+  fillColor?: string
+  [x: string]: unknown
+}
+
+const CheckmarkOutline: React.FunctionComponent<Props> = ({
+  width = '12',
+  height = '10',
+  fillColor = '#000000',
+  ...rest
+}) => (
+  <svg width={width} height={height} viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We don't need the `xmlns` prop since its in our custom component

Reply:
correct, it's only needed when SVGs are loaded from the server from another domain. This one will be inlined